astronomer/providers/sftp/example_dags/example_sftp.py

```diff
@@ -212,6 +213,16 @@ def terminate_instance(task_instance: "TaskInstance") -> None:
     )
 
 
+def check_dag_status(**kwargs: Any) -> None:
```

Comment:
As this is repeated in multiple DAGs, should we extract it to avoid duplication?

Reply:
I had this thought but have following next thoughts:
1. These are example DAGs and need to be self sufficient
2. We may not want to keep this as a util and release it as part of our providers

Owing to the above reasons, I guess the duplication should be okay.